appengine/cloudsql/README.md

```diff
@@ -35,51 +38,85 @@ use this service account to connect to your Cloud SQL instance locally.
 machine:
 
         ./cloud_sql_proxy \
-            -instances=[YOUR_INSTANCE_CONNECTION_NAME]=tcp:3306 \
+            -instances=[YOUR_INSTANCE_CONNECTION_NAME]=tcp:[PORT] \
             -credential_file=PATH_TO_YOUR_SERVICE_ACCOUNT_JSON_FILE
 
     where `[YOUR_INSTANCE_CONNECTION_NAME]` is the connection name of your
     instance on its Overview page in the Google Cloud Platform Console, or use
-    `[YOUR_PROJECT_ID]:[YOUR_REGION]:[YOUR_INSTANCE_NAME]`.
+    `[YOUR_PROJECT_ID]:[YOUR_REGION]:[YOUR_INSTANCE_NAME]`. If you're using
+    MySQL, `[PORT]` will be `3306`; for Postgres, it will be `5432`.
+
+### MySQL configuration
+To configure the sample for MySQL, use the following steps:
 
 1. Use the MySQL command line tools (or a management tool of your choice) to
 create a [new user][user] and [database][database] for your application:
 
         mysql -h 127.0.0.1 -P 3306 -u root -p
-          mysql> create database `YOUR_DATABASE`;
-          mysql> create user 'YOUR_USER'@'%' identified by 'PASSWORD';
-          mysql> grant all on YOUR_DATABASE.* to 'YOUR_USER'@'%';
+        mysql> create database `YOUR_DATABASE`;
+        mysql> create user 'YOUR_USER'@'%' identified by 'PASSWORD';
+        mysql> grant all on YOUR_DATABASE.* to 'YOUR_USER'@'%';
```

Comment:
Is this necessary still? The postgres docs below indicate this can all be done with the Cloud Console.